lua/gatekeeper/staticlib.lua

```diff
@@ -344,6 +344,16 @@ c = ffi.C
 
 local ifaces = require("if_map")
 
+function check_ifaces(front_ports, back_ports)
+	for i1, v1 in ipairs(front_ports) do
+		for i2, v2 in ipairs(back_ports) do
+			if ifaces[v1] == ifaces[v2] then
```

Comment:
Style: define a local variable for `ifaces[v1]` instead of computing it every time.

Reply:
Another important check that can be done here is to verify that the maps `ifaces[v1]` and `ifaces[v2]` exist. This may be better done while looking for duplicates at `front_ports` and `back_ports`.

Reply:
Ignore my comment above since it's already done in `init_iface()`.